### ISSUE TITLE
Is any way to customize the button content?

### ISSUE THREAD

**Issue by jvillane** [alexandrtovmach/react-microsoft-login#15]
I'm using this component but I can't managed to use for example the MicrosoftLogin children prop to do this.

Any recommendation will be appreciated.

Thanks in advance!

**Comment by alexandrtovmach** [comment]
It's in my [to do](https://github.com/alexandrtovmach/react-microsoft-login#to-do), but in the same time please check [this issue](https://github.com/alexandrtovmach/react-microsoft-login/issues/11)

**Comment by alexandrtovmach** [comment]
Available in [1.3.0 version](https://www.npmjs.com/package/react-microsoft-login/v/1.3.0)

```
<MicrosoftLogin
  clientId={clientId}
  authCallback={this.loginHandler}
>
  <button className="your-class">Custom button</button>
</MicrosoftLogin>
```
